firmware-binaries/test-cases/dna_port_e2_test/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2024 Google LLC
+#
+# SPDX-License-Identifier: CC0-1.0
+
+[package]
+name = "dna_port_e2_test"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+authors = ["Google LLC"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+riscv-rt = "0.11.0"
+bittide-sys = { path = "../../../firmware-support/bittide-sys", features = ["panic-handler"] }
```

Comment:
the `panic-handler` feature doesn't actually do anything anymore. It wasn't removed from the `Cargo.toml` file of `bittide-sys`, so it's not an error, but I'll do a pass removing it and all references to it soon, would be nice if it wasn't introduced if it doesn't do anything :sweat_smile: